compiler/dalgona/tests/GenH5RandomInputs.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import h5py as h5
+import numpy as np
+from circle.Model import Model
+import argparse
```

Comment:
(optional)
```suggestion
import h5py as h5
import numpy as np
import argparse

from circle.Model import Model
```

Reply:
Q) maybe new `compiler/dalgona` is not format checked for python files...
recent `onecc-docker` draft module didn't either...
@jinevening can you please check the format checker is working correctly?

Reply:
format checker works for "*.py" files. I've checked that format checker checks this file :)